test/ctest_helpers.cmake

```diff
@@ -236,16 +236,17 @@ macro(define_c_test)
             # Which while technically valid breaks assumptions in our testing utilities. Wrap the execution in powershell to avoid this.
             set(exec_wrapper "powershell.exe")
         endif()
-        if (C_TEST_SMOKE)
+        if (C_TEST_EXEC_SCRIPT)
             # csuite test comes with a smoke execution wrapper.
```

Comment:
Should this comment be reworded by excluding reference to 'smoke'?

Reply:
Fixed